.github/workflows/node.js.yml

```diff
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: 2021 Anders Rune Jensen
+#
+# SPDX-License-Identifier: Unlicense
+
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  licenses:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: REUSE Compliance Check
+        uses: fsfe/reuse-action@v1
```

Comment:
You sure you want to add this? We would have to add license headers on all files